App.tsx

```diff
@@ -27,14 +27,24 @@ const switchNavigator = createSwitchNavigator({
       },
     },
   }),
-  mainFlow: createMaterialBottomTabNavigator({
-    caseContactListFlow: createStackNavigator({
-      CaseContactList: CaseContactListScreen,
-      CaseContactDetail: CaseContactDetailScreen,
-    }),
-    CaseContactCreate: CaseContactCreateScreen,
-    Account: AccountScreen,
+  // mainFlow: createMaterialBottomTabNavigator({
```

Comment:
@michaelruhl what was the reason for commenting out the bottom tab navigator?

Reply:
It was causing issues with the styling.

Reply:
ok, let's not touch the navigation structure and revert it back to before.
The reason is in the future we will need to refactor the nav structure to use v6 of react navigation anyways (current version is 2 which is no longer maintained)
Honestly, this is my fault - I should've used v6 in the beginning, but I was following a tutorial using v2 and didn't realize how outdated the version was.
So, let's not waste time on styling the nav.

tl;dr I messed up and so let's just not touch the navigation structure as of now
@michaelruhl